### ISSUE TITLE
Crash, yet again

### ISSUE THREAD

**Issue by The96rf** [MobilePorting/FNF-JS-Engine-Mobile#6]
### Describe your bug here. If you are modding, try testing your bug in a clean version of the engine instead. Also, if you're using an older version of JSE, please try the latest version/action build. Also, be sure to check the pinned 'JS Engine: Known Issues' issue, to check if your issue hasn't already been found!

Hello, you guys might remember me from the start up glitch a long time ago, but there is yet ANOTHER one in the 1.33.0 update, it works fine on my other device (with some lag for some reason) but when it loads the new files THIS happens (BTW this is the second log because the first log is WAY too long)

### Command Prompt/Terminal/Crash logs (if existing)

```bash
Could not create directory:logs

/opt/hostedtoolcache/haxe/4.3.6/x64/std/cpp/_std/sys/

FileSystem.createDirectory() [line 88]

mobile/CopyState.update() [line 91]

flixel/FixState.tryUpdate() [line 239]

flixel/FlxGame.update() (line 749]

flixel/FlxGame.step() [line 679]

flixel/FlxGame.onEnterFrame() [line 552]

openfl/events/EventDispatcher._dispatchEvent() [line 426]

openfl/display/DisplayObject._dispatch() [line 1398]

i thought it was invincible
```

### Have you identified any steps to reproduce the bug? If so, please describe them below in as much detail as possible. Use images if possible.

```bash
No
```

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Android

### Did you edit anything in this build? If so, mention or summarize your changes.

No

### If you use 1.12.0 or earlier, did you have Optimized Chart Loading turned on?

None

### Did you check for any similar issues to what you're reporting? Check, then come back here. If there is a similar issue, then do not report the issue, otherwise it will be marked as a duplicate.

No

**Comment by mcagabe19** [comment]
lack of permissions, u need grant them

**Comment by The96rf** [comment]
I did

**Comment by mcagabe19** [comment]
if u use .JSEngine check if all files permission granted for js else use EXTERNAL_DATA or EXTERNAL_OBB with fully reinstall, I can't do anything to 13+ permission issues.

**Comment by mcagabe19** [comment]
fixed on 1.13.3
